gcp_variant_transforms/vcf_to_bq_preprocess.py

```diff
@@ -107,7 +107,8 @@ def run(argv=None):
                                                all_patterns,
                                                pipeline_mode,
                                                allow_malformed_records=True,
-                                               pre_infer_headers=True)
+                                               pre_infer_headers=True,
+                                               use_1_based_coordinate=False)
```

Comment:
Is this argument related to preprocess logic? To me, similar to `sample_name_encoding` this arg is irrelevant to preprocessor.
If that's the case let's not add this line.

Reply:
Ahh, yeah, I had integration test failures, was trying different things. Removed.